src/mca/pstrg/lustre/pstrg_lustre.c

```diff
@@ -156,6 +156,7 @@ static pmix_status_t query(pmix_query_t queries[], size_t nqueries, pmix_list_t
                 }
             }
 
+            pmix_output(0, "SILENCE WARNINGS %u %u", uid, gid);
```

Comment:
I think the cause of the CI failure is this debugging line that was left in place.